events/janus_gelfevh.c

```diff
@@ -0,0 +1,476 @@
+/*! \file   janus_gelfevh.c
+ * \author Lorenzo Miniero <lorenzo@meetecho.com>
+ * \copyright GNU General Public License v3
+ * \brief  Janus GelfEventHandler plugin
+ * \details  This is a trivial event handler plugin for Janus, which is only
+ * there to showcase how you can handle an event coming from the Janus core
+ * or one of the plugins. This specific plugin forwards every event it receives
+ * to a web server via an UDP request, using libcurl.
```

Comment:
The description seems incorrect (possibly a copy of the sample event handler?). You should clarify exactly what this handler is supposed to talk to, possibly with links to Graylog, the GELF format, etc. If multiple protocols are supported (I think you mentioned UDP and TCP?) and there are custom features (compression?) they should be mentioned too. Whatever you write here is converted to a documentation web page with Doxygen, and eventually pushed to the docs online.

Reply:
Just noticed the other handlers don't have a good description either... I'll fix those to give a good example :smile: